docs: clarify rationales for oauth authentication (#3538)
kubeshop/testkube: docs/docs/getting-started/index.md

You can also see the full installation video from our product experts: [Testkube Installation Video](https://www.youtube.com/watch?v=bjQboi3Etys):

<iframe width="100%" height="315" src="https://www.youtube.com/embed/ynzEkOUhxKk" title="YouTube Tutorial: Getting started with Testing in Kubernetes Using Testkube" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

> As a summary, you now have a ready testkube setup in your kubernetes cluster, ready to discover the testkube features:

Reviewer: ```suggestion
In summary, you now have Testkube setup in your Kubernetes cluster, ready to discover the Testkube features:
```

- the testkube cli enables to locally port forward to the kube dashboard deployment

> - the testkube cli enables to interact with testkube server components through k8s CRs or REST api calls made through the [kube apiserver proxy](https://kubernetes.io/docs/concepts/cluster-administration/proxies/)

Reviewer: ```suggestion
- The Testkube CLI allows you to interact with the Testkube server components through k8s CRs or REST API calls made through the [kube apiserver proxy](https://kubernetes.io/docs/concepts/cluster-administration/proxies/).
```

> As usage of testkube grows within your team, you may choose to:

Reviewer: ```suggestion
As usage of Testkube grows within your team, you may choose to:
```

> * leverage [managed testkube cloud](../testkube-cloud/intro.md)

Reviewer: ```suggestion
* Leverage [managed Testkube cloud](../testkube-cloud/intro.md).
```

> * [getting to production](../guides/going-to-production) with your own testkube installation 

Reviewer: ```suggestion
* [Move to production](../guides/going-to-production) with your own Testkube installation.
```
